eisbuk-admin/src/store/selectors/__testData__/customers.ts

```diff
@@ -0,0 +1,24 @@
+import { LocalStore } from "@/types/store";
+
+import { createTestStore } from "@/__testUtils__/firestore";
+import { saul, walt, jian } from "@/__testData__/customers";
+
+/**
+ * A customers store input
+ */
+const customers: LocalStore["firestore"]["data"]["customers"] = {
+  [saul.secretKey]: saul,
+  [jian.secretKey]: jian,
+  [walt.secretKey]: walt,
+};
+
+export const expectedCustomersBirthdays = {
+  "01/01": [saul, jian, walt],
```

Comment:
I wouldn't use the `mm/dd` format (or vice versa), but rather the standard ISO `mm-dd` as keys. The conversion for displaying should be done at component level: if you want to display `31.10` instead of `10-31`, but still use `10-31` to process/key/query the data.